src/components/drawer/ComponentDetailPanel.vue

```diff
@@ -70,6 +71,10 @@ const selectionSubscription = ComponentSelectionEvent.subscribe((newVal) => {
 function closeDrawer() {
   ComponentSelectionEvent.next(null);
 }
+// dummy function for now, will probably recieve more logic
+function save() {
+  closeDrawer();
```

Comment:
may be you can send event like ComponentEvent.next('update', component);

Reply:
added a new subject for component update